examples/gallery/symbols/text_symbols.py

```diff
@@ -15,9 +15,6 @@
 as **+j**\ *justify* to change the justification [Default is CM]. The fill
 color of the text symbols can be set with the ``fill`` parameter, and the
 outline width can be customized with the ``pen`` parameter.
-For all supported octal codes and fonts see the GMT Technical Reference
-:gmt-docs:`reference/octal-codes.html` and
-:gmt-docs:`reference/postscript-fonts.html`.
```

Comment:
I prefer to remove this sentence because this gallery example should focus on "Typesetting text like symbols".

Reply:
Maybe we can reduce the sentence to:

```
For all supported fonts see :doc:`/techref/fonts`.
```
and move it up:
```
(less than 256 characters). Optionally, you can append
**+f**\ *font,outlinecolor* to select a particular font [Default is
:gmt-term:`FONT_ANNOT_PRIMARY`] and outline color [Default is black] as well
as **+j**\ *justify* to change the justification [Default is CM]. For all supported
fonts see :doc:`/techref/fonts`. The fill color of the text symbols can be set with
the ``fill`` parameter, and the outline width can be customized with the ``pen``
parameter.
```

Reply:
Done in 9132a3f79a7f6529b25fcad1584a0d5302eda1a7.